src/helper.js

```diff
@@ -90,6 +91,10 @@ function getUserLogsPath(name, network) {
 	return path.join(this.HOME, "logs", name, network);
 }
 
+function getStoragePath() {
+	return path.join(this.HOME, "storage");
```

Comment:
I think we will want to improve that in the near future. .lounge mostly has config stuff right now so I personally have it in /etc/ for now. It's not ideal with user logs, but file storage makes this even more inappropriate.
Any suggestions?